src/runtime/virtcontainers/pkg/oci/utils.go

```diff
@@ -616,6 +617,15 @@ func addHypervisorMemoryOverrides(ocispec specs.Spec, sbConfig *vc.SandboxConfig
 		return err
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.EnableGuestSwap]; ok {
```

Comment:
Setting bool values can be changed to like lines 614-618.

Reply:
Fixed.